Car.cpp

```diff
@@ -12,6 +12,6 @@ void Car::closeWindows()
 
 void Car::tryToEvade()
 {
-    setSpeed(0);
-    std::cout << name << ": hello officer, what seems to be the problem?" << std::endl;
+    setSpeed(90);
+    std::cout << name << ": kiss my shining metall bumper :-*" << std::endl;
 }
```

Comment:
lol
You didn't need to change this, but it's fine.
However, line 17 will cause a warning.
fix the warning
- [ ] revise